### DISCUSSION TITLE
Batch size is not implemented?

### DISCUSSION THREAD

**Discussion by tsarikahin** [lululxvi/deepxde#1255] [Q&A]
Dear LuLu @lululxvi,

I was wondering if the `batch_size` in the `train` method is not implemented or if it does not work. It says: "If you solve PDEs via dde.data.PDE or dde.data.TimePDE, do not use batch_size, and instead use [dde.callbacks.PDEPointResampler](https://deepxde.readthedocs.io/en/latest/modules/deepxde.html#deepxde.callbacks.PDEPointResampler)"

Imagine that I must use `batch_size`.

I would really appreciate it if you could explain why you suggest using the callback `PDEPointResampler`.

Can you also confirm if `batch_size` uses `train_x_all` in case it is implemented?

Best,
Tarik

**Comment by lululxvi** [comment]
https://deepxde.readthedocs.io/en/latest/modules/deepxde.html#deepxde.callbacks.PDEPointResampler works the same as batch size.
